lib/testing-library-detection.ts

```diff
@@ -0,0 +1,52 @@
+import { TSESTree } from '@typescript-eslint/experimental-utils';
+
+/**
+ * Enhances a given rule with helpers to detect Testing Library utils.
+ */
+function testingLibraryDetection(rule: any) {
+  return (context: any, options: any) => {
+    let isImportingTestingLibrary = false;
+
+    // TODO: init here options based on shared ESLint config
+
+    // helpers for Testing Library detection
+    const helpers = {
+      getIsImportingTestingLibrary() {
+        return isImportingTestingLibrary;
+      },
+    };
+
+    // instructions for Testing Library detection
+    const detectionInstructions = {
+      ImportDeclaration(node: TSESTree.ImportDeclaration) {
+        isImportingTestingLibrary = /testing-library/g.test(
+          node.source.value as string
+        );
+      },
+    };
+
+    // update given rule to inject Testing Library detection
+    const ruleInstructions = rule(context, options, helpers);
```

Comment:
Is this correct? If I understand correctly, this comes from the parameter, which in line 48 is received

but in `lib/rules/no-node-access.ts` you define

```ts
create: detect((context: any, _: any, helpers: any) => {
// code 
}
```

should that ☝️  be 

```ts
create: detect((context: any, _: any, options: any, helpers: any) => {
// code 
}
```

?